pom.xml

```
@@ -49,4 +49,70 @@
         <module>choco</module>
     </modules>
 
+    <profiles>
+        <profile>
+            <id>Enforce-Java-11</id>
```

Could you please change this to:
```suggestion
            <id>enforce-jdk11</id>
```

Thanks! :)